tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""HerePY test package initialization"""
```

Comment:
I think adding `__init__.py` file is optional on this case and tests already running as expected I prefer not to add that file.

Reply:
You are right. It is redundant. I've just removed it. 

Nevertheless, it was the first workaround I found for running `py.test -v` from the repo directory on my environment.

After doing a bit of research I found [this StackOverflow entry](https://stackoverflow.com/questions/10253826/path-issue-with-pytest-importerror-no-module-named-yadayadayada).

As far as I understand, `pytest` tends to handle `sys.path` differently depending on the OS. I'm running over Windows 10, and `pytest` won't find the `herepy` module, since it would try to resolve it from inside the tests directory.

If you are running over macOS, a way to replicate the issue is by running a `pytest -v` inside a Linux container.

My suggestion is to follow the answer presented by hoefling, which I've already tested locally.

Looking forward to your feedback.

Reply:
Good point thanks for letting me know then only required change would be `HerePy` instead of `HerePY`.